### PR TITLE
fix: asset modal showing only image-based assets

### PR DESCRIPTION
**Pull request by KaWaite**
closes reearth/reearth#260

## What I've done
- Fix issue where file assets weren't selectable for things like 3d model primitives
- (Re)Add file icon for files
- Redo file upload restrictions

Note: asset modal now doesn't support limiting shown assets based on whether it is an image or file (at the code level).

### CODE REVIEW

File: src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts [hunk 1]
```diff
@@ -6,6 +6,10 @@ export type SortType = "date" | "name" | "size";
 
 export type LayoutTypes = "medium" | "small" | "list";
 
+export const fileFormats = ".kml,.czml,.json,.gltf,.glb";
```
Comment (rot1024) [inline src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts:9]:
```suggestion
export const fileFormats = ".kml,.czml,.topojson,.geojson,.json,.gltf,.glb";
```

File: src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts [hunk 2]
```diff
@@ -6,6 +6,10 @@ export type SortType = "date" | "name" | "size";
...
+
+export const imageFormats = ".jpg,.jpeg,.png,.gif,.svg,.tiff";
```
Comment (rot1024) [inline src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts:11]:
```suggestion
export const imageFormats = ".jpg,.jpeg,.png,.gif,.svg,.tiff,.webp";
```

File: src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts [hunk 3]
```diff
@@ -74,7 +76,7 @@ export default ({
       : "filterTime";
 
   const handleFileSelect = useFileInput(files => onCreateAssets?.(files), {
-    accept,
+    accept: imageFormats + "," + fileFormats,
```
Comment (rot1024) [inline src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts:79]: How is video formats?

**Review by rot1024 — CHANGES_REQUESTED** [review]

File: src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts (hunk 1 above)
Comment (KaWaite) [reply src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts:9]: Ah I had those but when I googled those formats it seemed like they were just json files with specific formatting. I'll re-add those!

File: src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts (hunk 3 above)
Comment (rot1024) [reply src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts:79]: Ah videos are always external, ok.

**Review by rot1024 — APPROVED** [review]

**Pull request merged**